package.json

```diff
@@ -94,15 +94,18 @@
 		"@prisma/client": "3.10.0",
 		"@remix-run/react": "1.2.3",
 		"@remix-run/serve": "1.2.3",
-		"argon2": "0.28.4",
+		"@remix-validated-form/with-zod": "^2.0.1",
+		"bcryptjs": "^2.4.3",
 		"framer-motion": "6.2.8",
 		"lodash": "4.17.21",
 		"react": "17.0.2",
 		"react-dom": "17.0.2",
 		"react-icons": "4.3.1",
 		"react-table": "7.7.0",
 		"remix": "1.2.3",
-		"sharp": "0.30.1"
+		"remix-validated-form": "^4.1.8",
+		"sharp": "0.30.1",
+		"zod-form-data": "^1.2.0"
```

Comment:
It would be more safe to install `zod` as a direct dependency as well

Reply:
Ok